js/canvasEditor.js

```diff
@@ -154,6 +154,21 @@ CanvasEditor.prototype.cropEditorCanvas = function(x, y, w, h, callback) {
   this.cropCanvas(this.editorCanvas, x, y, w, h, callback);
 };
 
+CanvasEditor.prototype.notInBorders = function(coords, mousePos) {
+  var $container = $(this.editorCanvas).parent();
+  var containerSize = {
+    height: $container.innerHeight(),
+    width: $container.innerWidth()
+  };
+  var mouseX = mousePos.pageX - $container.offset().left;
+  var mouseY = mousePos.pageY - $container.offset().top;
+
+  return ((coords.x === 0 && mouseX < 0) || 
+       (coords.y === 0 &&  mouseY < 0) || 
+       (coords.x2 === containerSize.width && mouseX > containerSize.width) || 
+       (mouseY > containerSize.height && coords.y2 === containerSize.height))
```

Comment:
```
  return (coords.x === 0 && mouseX < 0)
    || (coords.y === 0 &&  mouseY < 0)
    || (coords.x2 === containerSize.width && mouseX > containerSize.width)
    || (mouseY > containerSize.height && coords.y2 === containerSize.height);
```

- Logic operator goes in the beginning of the new line, not the end of the previous line
- Missing semicolon
- Outermost `()` are unnecessary

Reply:
@tonytlwu 
Updated return statement of the notInBorders function